### PR TITLE
Bug 1404461 - delete UI from login site

### PR DESCRIPTION
**Pull request by djmitche**
This is the highlight of my year, I think.

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -5,7 +5,6 @@ Defaults:
     authorizers:
```
Comment (helfi92) [inline config.yml]: Is https://github.com/djmitche/taskcluster-login/blob/ab01f586c6384a1f67db90c8caa7d636817a002b/config.yml#L3 still needed:

```
authenticators:
  - auth0
```

File: src/main.js [hunk 2]
```diff
@@ -1,26 +1,19 @@
-const express = require('express');
```
Comment (helfi92) [inline src/main.js]: Have the references to  `server.js` been updated to `main.js`? For example, I still see `test/helper.js` have: `const load = require('../src/server');`.

File: test/lint.js [hunk 3]
```diff
@@ -1,10 +0,0 @@
-var lint = require('mocha-eslint');
-
-var paths = [
-  'src/*.js',
-  'src/*/*.js',
```
Comment (helfi92) [inline test/lint.js]: Was this file not used before? I see that `package.json` has a `lint` command:

`"lint": "eslint src/*.js test/*.js"`

I think we should add 'src/*/*.js` and run `yarn lint` to make sure linting is still clean.

File: package.json [hunk 4]
```diff
@@ -16,39 +16,29 @@
   },
   "dependencies": {
     "auth0-js": "^8.12.0",
-    "body-parser": "^1.18.2",
     "browserid-verify": "^0.1.2",
```
Comment (helfi92) [inline package.json]: `browserid-verify` doesn't seem to be used.

File: src/main.js [hunk 5]
```diff
@@ -130,92 +99,26 @@ let load = loader({
...
-          auth0_domain: cfg.auth0.domain,
-          auth0_client_id: cfg.auth0.clientId,
-        });
+      let app = App({
+        port: cfg.server.port,
+        publicUrl: cfg.server.publicUrl,
+        env: cfg.server.env,
+        forceSSL: cfg.server.forceSSL,
+        trustProxy: cfg.server.trustProxy,
+        rootDocsLink: true, // doesn't work?
```
Comment (helfi92) [inline src/main.js]: It doesn't work?

Comment (djmitche) [reply src/main.js]: it does -- I've removed the comment now.

**Pull request merged**